src/pal/src/loader/module.cpp

```diff
@@ -676,7 +676,7 @@ PAL_RegisterModule(
 {
     HINSTANCE hinstance = nullptr;
 
-    int err = PAL_InitializeDLL();
+    int err = PAL_InitializeDLLWithoutSEH();
```

Comment:
I am not fully understanding the suggestion, so maybe it would work better if I toss out all the scenarios that are important so we can make sure they aren't a problem:

1. rc2 dbgshim loading RTM DAC/DBI. This will be pretty painful if it breaks, so be careful if DAC/DBI expect certain behavior.
2. VS + new PAL loading rc2 dbgshim. We are stuck on the rc2 dbgshim until right around RTM. But we could make changes to how we load dbgshim if necessary, so this isn't a big deal as long as we know what we need to change when we update our PAL.
3. VS copy of PAL loading another VS dll using LoadLibrary -- currently we have one VS dll that has an instance of the PAL. That dll will load other dlls via LoadLibrary, these new dlls will expect to have their DllMain called, but we can do that in some other way if required. Again, we just need to know what to do when we next take in a new PAL.